static/docs/commands-reference/commit.md

```diff
@@ -142,8 +142,8 @@ make is adjusting a parameter to `CountVectorizer` in that script. Namely,
 adjusting the `max_features` option in this line changes the resulting model:
 
 ```python
-    bag_of_words = CountVectorizer(stop_words='english',
-                max_features=6000, ngram_range=(1, 2))
+bag_of_words = CountVectorizer(stop_words='english',
```

Comment:
@shcheklein is the four space removal applicable for code blocks with python as well?
I thought it was only applicable for dvc code blocks in the PR https://github.com/iterative/dvc.org/commit/ab7482648d820a57a2b3c73c53180db708c9b5a8

Reply:
![Screenshot (20)](https://user-images.githubusercontent.com/16274905/57966608-12ca6f00-7972-11e9-916f-002f72cb4099.png)
This is the one with four space indentation in the python code block.

![Screenshot (21)](https://user-images.githubusercontent.com/16274905/57966610-1eb63100-7972-11e9-99c2-2646622a5327.png)
And, this is without four space indentation on the same block. 

Imho, python code blocks without four space indentation look great.

Reply:
It's not about how indentation looks in markdown which I am worried, I am worried how it will render in docs engine. Currently, the docs engine pads/indents only for DVC code blocks (`dvc`)

Reply:
Then, I can close this pull request, and make a new pull only for DVC blocks?

Reply:
Applies to all blocks. Thanks @kurianbenoy for reviewing this!